integration-tests/test-commons/src/main/java/com/github/mcollovati/quarkus/testing/AbstractTest.java

```diff
@@ -49,6 +49,7 @@ void setup() {
             Configuration.headless = runHeadless();
             System.setProperty("chromeoptions.args", "--remote-allow-origins=*");
         }
+        Configuration.fastSetValue = true;
```

Comment:
With this feature when you set a value on an input element (e.g. a text field) the value is set using javascript and then firing the related events.
Selenium by default send all character separately and this makes it difficult to test for example the grid filter because it is slower and some events gets lost

Reply:
I understand, thanks for clarification